ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.view.js

```diff
@@ -61,46 +73,88 @@ module.exports = Marionette.LayoutView.extend({
-    let modelJSON = this.model.toJSON()
-    let viewData = {
-      target: modelJSON.target,
-      lat: modelJSON.mouseLat,
-      lon: modelJSON.mouseLon,
-    }
-    switch (getCoordinateFormat()) {
+  getDisplayComponent: function() {
+    const coordinateFormat = getCoordinateFormat()
+    switch (coordinateFormat) {
       case 'degrees':
-        viewData.lat =
-          typeof viewData.lat === 'undefined'
-            ? viewData.lat
-            : mtgeo.toLat(viewData.lat)
-        viewData.lon =
-          typeof viewData.lon === 'undefined'
-            ? viewData.lon
-            : mtgeo.toLon(viewData.lon)
-        break
+        return this.degreesComponent()
       case 'decimal':
-        viewData.lat =
-          typeof viewData.lat === 'undefined'
-            ? viewData.lat
-            : `${leftPad(Math.floor(viewData.lat), 3, ' ')}.${Math.abs(
-                viewData.lat % 1
-              )
+        return this.decimalComponent()
+      case 'mgrs':
+        return this.mgrsComponent()
+      case 'utm':
+        return this.utmComponent()
+    }
+    throw 'Unrecognized coordinate format value [' + coordinateFormat + ']'
+  },
+  degreesComponent() {
+    const lat = this.model.get('mouseLat')
+    const lon = this.model.get('mouseLon')
+    return (
+      <div className="info-coordinates">
+        <span className="coordinate-lat">
+          {typeof lat === 'undefined' ? lat : mtgeo.toLat(lat)}
+        </span>{' '}
+        <span className="coordinate-lon">
+          {typeof lon === 'undefined' ? lon : mtgeo.toLon(lon)}
+        </span>
+      </div>
+    )
+  },
+  decimalComponent() {
+    const lat = this.model.get('mouseLat')
+    const lon = this.model.get('mouseLon')
+    return (
+      <div className="info-coordinates">
+        <span className="coordinate-lat">
+          {typeof lat === 'undefined'
+            ? lat
+            : `${leftPad(Math.floor(lat), 3, ' ')}.${Math.abs(lat % 1)
                 .toFixed(6)
                 .toString()
-                .slice(2)}`
-        viewData.lon =
-          typeof viewData.lon === 'undefined'
-            ? viewData.lon
-            : `${leftPad(Math.floor(viewData.lon), 4, ' ')}.${Math.abs(
-                viewData.lon % 1
-              )
+                .slice(2)}`}
+        </span>{' '}
+        <span className="coordinate-lon">
+          {typeof lon === 'undefined'
+            ? lon
+            : `${leftPad(Math.floor(lon), 4, ' ')}.${Math.abs(lon % 1)
                 .toFixed(6)
                 .toString()
-                .slice(2)}`
-        break
-    }
-    return viewData
+                .slice(2)}`}
+        </span>
+      </div>
+    )
+  },
+  mgrsComponent() {
+    const lat = this.model.get('mouseLat')
+    const lon = this.model.get('mouseLon')
+    return (
+      <div className="info-coordinates">
+        {/* Possibly use a different class name; one not coupled to lat/lon? */}
+        <span className="coordinate-lat">
```

Comment:
This `coordinate-lat` class should be renamed since it now describes more coordinates besides latitude.

Reply:
Was able to remove it entirely 👍